src/commands/utility/calendar.js

```diff
@@ -0,0 +1,29 @@
+const { SlashCommandBuilder } = require('discord.js');
+const {
+  scheduleEventNotification,
+} = require('../../cron/schedule-google-calendar');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('calendar')
+    .setDescription('Replies with Pong!'),
+  async execute() {
+    try {
+      const events2 = [
+        {
+          summary: 'Reunión importante',
+          start: { dateTime: '2024-11-26T10:56:00Z' }, // Hora UTC
```

Comment:
Use English

Reply:
This command is not necessary. It was used only for testing